### DISCUSSION TITLE
关于和Bark的配合（寻求帮助）

### DISCUSSION THREAD

**Discussion by graceall** [shuax/MouseInc.Settings#166] [Q&A]
你好，我希望通过mouseinc实现一些功能，但不是很会使用自定义动作，希望请教一下
Bark是一款开源的可以使iPhone获得PC剪贴板内容推送的软件
它的原理是，通过电脑访问_https://api.day.app/“移动设备码”/“剪贴板内容”_这个网址
由APP向手机推送“剪贴板内容”，获得PC和iPhone的联动
现在我已经利用mouseInc实现了通过鼠标手势将所选内容推送到手机
但这个过程会打开一个浏览器窗口，我希望能实现访问网址时不出现浏览器窗口
或者出现浏览器窗口后能自动关闭这个窗口，请问可以实现吗？如果可以实现，应该怎么做呢？
感谢向我提供帮助的人。

**Comment by LCC456** [comment]
修改手势动作指令：
`
[
	["Execute", "curl https://api.day.app/移动设备码/剪贴板内容"]
]
`

![image](https://user-images.githubusercontent.com/32818020/135371373-996d272c-cd7c-44e8-b8b0-554863a99fdf.png)

**Reply by graceall** [reply]
完全达到目的，非常感谢
